timemachine/cpp/src/context.cu

```diff
@@ -19,7 +19,6 @@ Context::Context(
     d_x_t_ = gpuErrchkCudaMallocAndCopy(x_0, N * 3);
     d_v_t_ = gpuErrchkCudaMallocAndCopy(v_0, N * 3);
     d_box_t_ = gpuErrchkCudaMallocAndCopy(box_0, 3 * 3);
-    gpuErrchk(cudaMalloc(&d_du_dl_buffer_, N * sizeof(*d_du_dl_buffer_)));
     gpuErrchk(cudaMalloc(&d_u_buffer_, N * sizeof(*d_u_buffer_)));
 
     unsigned long long *d_in_tmp = nullptr;  // dummy
```

Comment:
nit : unsure -- comment on L27 references du_dl but I think describes code that's still needed regardless of du_dl?

Reply:
Correct, we use `d_sum_storage_` and `d_sum_storage_bytes_` on L156 (in the reduction of per-atom energies). Will update the comment.

Reply:
Updated comment in 440ba99